node_shell.py

```diff
@@ -559,7 +586,7 @@ def do_extrakeys(self, arg):  # pylint: disable=unused-argument
         for key in sorted(extras_keys):
             self.poutput('- {}'.format(key))
 
-    @needs_new_node
+    @needs_new_node(always=True)
     @with_default_argparse
```

Comment:
For attributes you don't need always=True, as they cannot be changed after sealing, do you agree?